tools/publish-aos.js

```diff
@@ -41,9 +41,12 @@ async function main() {
     'Changelog',
     `# Changelog
 
+
 ### Fixed
 
-- Updated Credit and Debit notices to align with token spec as outlined in: [https://github.com/permaweb/aos/blob/main/blueprints/token.lua]
```

Comment:
Not sure if it was intentional to remove this note or to add a new bullet point for fixed items.